Include/internal/pycore_pymem.h

```diff
@@ -88,7 +88,8 @@ extern wchar_t *_PyMem_DefaultRawWcsdup(const wchar_t *str);
 extern int _PyMem_DebugEnabled(void);
 
 // Enqueue a pointer to be freed possibly after some delay.
-extern void _PyMem_FreeDelayed(void *ptr, size_t size);
+# Exported for 'array' shared extension.
```

Comment:
Oops, wrong comment type:
```suggestion
// Exported for 'array' shared extension.
```